Gemfile

```diff
@@ -110,6 +110,8 @@ group :development, :test do
   gem 'spring-commands-cucumber'
   gem 'spring-commands-rspec'
   gem 'table_print'
+  gem 'ed25519'
+  gem 'bcrypt_pbkdf'
```

Comment:
what are these gems? why do we need them?

Reply:
Required when you ssh with a private key.

Reply:
ok i see that they are under `:development, :test` so it's fine